test/data/javascript/tests.js

```diff
@@ -4546,6 +4546,15 @@ exports.test_data = {
             '    }',
             ')'
           ]
+        },
+        {
+          comment: 'Issue #1950: Do not remove whitespace after number - test scenario: number before a dot',
```

Comment:
I'm getting a test failed error here even when I see that expected and ouput are the same. @bitwiseman, any idea what could I be missing?
`Testing js-beautifier with node.js CommonJS (src/index)...
==== Destructured and related ============================================================
---- input -------
1000000000000001000 .toFixed(0)!==1000000000000001024
---- expected ----
1000000000000000100 .toFixed(0) !== 1000000000000001024
---- output ------
1000000000000001000 .toFixed(0) !== 1000000000000001024
---- expected-ws ------
1000000000000000100_.toFixed(0)_!==_1000000000000001024
---- output-ws ------
1000000000000001000_.toFixed(0)_!==_1000000000000001024
================================================================`